tests/Tests/500__jupyterhub/test-versions.robot

```diff
@@ -24,74 +27,69 @@ Open JupyterHub Spawner Page
     [Documentation]    Verifies libraries in Tensorflow image
     [Tags]    Sanity
-    ...       ODS-204  ODS-205  ODS-206  ODS-207
-    @{additional_libs} =    Create List
-    Append To List    ${additional_libs}    JupyterLab v3.2    Notebook v6.4
-    ${status} =  Verify Libraries In Base Image    tensorflow    ${additional_libs}
-    Append To List  ${status_list}  ${status}
-    Run Keyword If  '${status}' == 'FAIL'  Fail  Shown and installed libraries for tensorflow image do not match
+    ...       ODS-204    ODS-205    ODS-206    ODS-207
+    Verify List Of Libraries In Image    tensorflow    JupyterLab v3.2    Notebook v6.4    JupyterLab-git v0.30
 
 Verify All Images And Spawner
     [Documentation]    Verifies that all images have the correct libraries
     [Tags]    Sanity
-    ...       ODS-340
-    List Should Not Contain Value  ${status_list}  FAIL
-    ${length} =  Get Length  ${status_list}
-    Should Be Equal As Integers  ${length}  4
-    Log To Console  ${status_list}
+    ...       ODS-340    ODS-452
+    List Should Not Contain Value    ${status_list}    FAIL
+    ${length} =    Get Length    ${status_list}
+    Should Be Equal As Integers    ${length}    4
+    Log To Console    ${status_list}
 
 
 *** Keywords ***
-Verify Libraries In Base Image  # robocop: disable
+Verify Libraries In Base Image    # robocop: disable
     [Documentation]    Fetches library versions from JH spawner and checks
     ...    they match the installed versions.
-    [Arguments]  ${img}  ${additional_libs}
-    @{list} =  Create List
-    ${text} =  Fetch Image Description Info  ${img}
-    Append To List  ${list}  ${text}
-    ${tmp} =  Fetch Image Tooltip Info  ${img}
-    ${list} =  Combine Lists  ${list}  ${tmp}  # robocop: disable
-    ${list} =  Combine Lists  ${list}  ${additional_libs}
-    Log  ${list}
-    Spawn Notebook With Arguments  image=${img}
-    ${status} =  Check Versions In JupyterLab  ${list}
+    [Arguments]    ${img}    ${additional_libs}
+    @{list} =    Create List
+    ${text} =    Fetch Image Description Info    ${img}
+    Append To List    ${list}    ${text}
+    ${tmp} =    Fetch Image Tooltip Info    ${img}
+    ${list} =    Combine Lists    ${list}    ${tmp}    # robocop: disable
+    ${list} =    Combine Lists    ${list}    ${additional_libs}
+    Log    ${list}
+    Spawn Notebook With Arguments    image=${img}
+    ${status} =    Check Versions In JupyterLab    ${list}
     Clean Up Server
     Stop JupyterLab Notebook Server
-    Go To  ${ODH_DASHBOARD_URL}
+    Go To    ${ODH_DASHBOARD_URL}
     Wait For RHODS Dashboard To Load
     Launch JupyterHub From RHODS Dashboard Link
     Fix Spawner Status
     Wait Until JupyterHub Spawner Is Ready
-    [Return]  ${status}
+    [Return]    ${status}
 
 Load Spawner Page
     [Documentation]    Suite Setup, loads JH Spawner
     Begin Web Test
     Launch JupyterHub Spawner From Dashboard
+
+Verify List Of Libraries In Image
```

Comment:
still not convinced about the naming here, but if no one else see the naming issue we could leave it as it is :) 
@jgarciao @lugi0

Reply:
I suggested that name to @pranavsuryaw, not sure what else to put there :/ Did you have any better ideas?

Reply:
based on the names they seem to be the same keyword, but I think we can leave it because they are not common keywords to use anywhere else or to be reported in docs I guess. I was struggling to to think of better names acutally